pkg/constants/env_variables.go

```diff
@@ -79,4 +79,8 @@ const (
 	// SkipTAPScopesValidationOnTanzuContext skips the TAP scopes validation on the token acquired while creating "tanzu"
 	// context using tanzu login or tanzu context create command
 	SkipTAPScopesValidationOnTanzuContext = "TANZU_CLI_SKIP_TAP_SCOPES_VALIDATION_ON_TANZU_CONTEXT"
+
+	// UseStableKubeContextNameForTanzuContext uses the stable kube context name associated with tanzu context.
+	// CLI would not change the context name when the TAP resource pointed by the CLI context is changed.
+	UseStableKubeContextNameForTanzuContext = "TANZU_CLI_USE_STABLE_KUBE_CONTEXT_NAME"
```

Comment:
Nit: Update this table in docs (https://github.com/vmware-tanzu/tanzu-cli/tree/main/docs/full#environment-variables-affecting-the-cli) to include this new environment variable.

_(Also, add `TANZU_CLI_SKIP_TAP_SCOPES_VALIDATION_ON_TANZU_CONTEXT` if it is missing)_

Reply:
Updated. Thanks